src/program/downloaders/shared.py

```diff
@@ -15,6 +17,89 @@
 if not VIDEO_EXTENSIONS:
     VIDEO_EXTENSIONS = DEFAULT_VIDEO_EXTENSIONS
 
+# Type aliases
+InfoHash = str # A torrent hash
+DebridTorrentId = str # Identifier issued by the debrid service for a torrent in their cache
```

Comment:
I threw these type aliases in here so it's clearer which IDs are which when implementing a downloader. I found it a bit easier than `str` everywhere - YMMV